scripts/portal.js

```diff
@@ -18,7 +18,7 @@ function Portal (sites) {
   function _directory (sites) {
     return `
     <ul>${sites.reduce( (acc, site, id) =>
-          `${acc}<li data-type='${site.type || ''}' id='${id}'><a href='${site.url}'>${_name(site)}</a></li>`
+          `${acc}<li ${site.type ? `data-type="${site.type}"` : ''} id='${id}'><a href='${site.url}'>${_name(site)}</a></li>`
```

Comment:
> data-type="${site.type}"
should i add escaping of possible double quotes in site-types? like:
``` js
`data-type="${site.type.replace(/"/g, '&quot;')}"`
```

Reply:
My initial opinion is no since none of the site types should have a quote in them.